about/faq.rst

```diff
@@ -183,6 +183,30 @@ free and open-source integration, consider starting the integration work yoursel
 Godot is not owned by one person; it belongs to the community, and it grows along
 with ambitious community contributors like you.
 
+Why does Godot use Vulkan or OpenGL instead of Direct3D?
+--------------------------------------------------------
+
+This question usually stems from OpenGL having a bad reputation in the 2000s for
+poor driver compatibility and lower performance. However, many popular
+OpenGL-based games have proven to be well-optimized in practice.
+
+OpenGL drivers have gotten much better over the recent years. While AMD graphics
+drivers on Windows have historically suffered from lower OpenGL performance,
+this has been remedied by Vulkan which is a lower-level API with fewer
+driver-specific quirks.
```

Comment:
I don't think this kind of background on how OpenGL used to be bad is useful here. We don't need to try to justify why users might ask this question, we just have to answer it.

IMO what users need to know is that Godot's aims for cross-platform compatibility and open standards first and foremost. OpenGL and Vulkan are the technologies that are both open and available (nearly) on all platforms, and therefore that's why we give them the priority. A game developed with Godot on Windows will run out of the box on Linux and macOS thanks to this design decision.